src/routes/blog/spyder-6-carlos/+page.md

```diff
@@ -0,0 +1,156 @@
+These changes added customizable default run presets for specific file types, making it easier to customize run behavior.
+In addition, the layout of that preference pane along with the per-file run config and the run config dialog are substantially simplified, cleaned up and polished, making the UI easier to use and more intuitive.
+
+![Before and after of Spyder's Run Config Per File dialog listing file execution settings, with the after version showing a simpler, collapsible and more refined layout](run-config-per-file.png "Carlos' UI improvements to the Run Config Per File dialog")
+
+
+## Insights going forward
+
+The Editor migration and the new remote client plugin were a fascinating set of stress tests for Spyder's plugin API, originally introduced in Spyder 5.
+They required implementing close integration with the IPython Console, but without affecting the already extensive connections between the console and several other plugins.
+
+Fortunately, our API passed the test really well, and we were able to have a working prototype of the Editor as a plugin for Linux and Mac in just four months.
+This means that we're in good shape to tackle the complex challenges that will come in 6.1 and beyond to add remote development capabilities to other Spyder plugins, such as the Editor and Projects.
+
+
+## What's next for Spyder?
+
+![Spyder-Env-Manager pane showing an environment selector and a list of installed package names, descriptions and versions as well as toolbar options to interact with them](spyder-env-manager-alpha.png "Working alpha version of the Spyder-Env-Manager plugin")
+
+Next year, I'll be working on more UI/UX improvements on several different fronts.
+
+I'll integrate the [Spyder-Env-Manager](https://github.com/spyder-ide/spyder-env-manager) plugin, developed by [Daniel Althviz](https://github.com/dalthviz) and [Juan-Sebsatian Bautista](https://github.com/jsbautista), with the remote development client.
+Spyder-Env-Manager allows you to create Conda environments as well as install and manage packages within them from within Spyder's graphical interface—no arcane commands or external GUIs, and mo separate install of Anaconda with our standalone installers.
+With the integration, you'll be able to create and manage Conda environments on remote machines just as easily as locally, without needing to install anything on the remote server.
+
+In addition, I'll be adding new preferences to allow you to easily associate your Spyder projects with Conda environments.
+That way, they can be automatically activated both for installing packages and running code, addressing the all too common issue of doing one or the other in the wrong environment without realizing.
+
+Finally, I'll improve the file/symbol switcher's look-and-feel and integrate it with other plugins.
+For example, you'll be able to type the IPython console to quickly open or switch to a specific console or Python environment.
```

Comment:
Yeah, the sentence syntax is quite complex, but the suggestion I find even more confusing since it also took a couple reads to parse and the intended meaning is ambiguous due to the ambiguity in the left operand to the final "or" due to the construction, as well as the very unbalanced length of the presumably intended operands. It reads something like

> For example, you'll be able to (type the IPython console you want to switch to,) or (quickly open through the switcher.)

Or, upon second reading, perhsp

> For example, you'll be able to type the IPython console you want to (switch to,) or (quickly open through the switcher.)

And after some more reading, I thought it might be

> For example, you'll be able to type the IPython console you want to (switch to,) or (quickly open) through the switcher.

However, none of these entirely make sense.

Taking a better stab at it:

```suggestion
For example, you'll be to quickly open or switch to a specific console just by typing its name.
```